comtypes/test/test_excel.py

```diff
@@ -4,76 +4,76 @@
 import datetime
 import unittest
 
-import comtypes.test
-from comtypes.client import CreateObject
+from comtypes.client import CreateObject, GetModule
 
-comtypes.test.requires("ui")
+################################################################
+#
+# TODO:
+#
+# It seems bad that only external test like this
+# can verify the behavior of `comtypes` implementation.
+# Find a different built-in win32 API to use.
+#
+################################################################
 
+try:
+    GetModule(("{00020813-0000-0000-C000-000000000046}",))  # Excel libUUID
+    from comtypes.gen.Excel import xlRangeValueDefault
+    IMPORT_FAILED = False
+except (ImportError, OSError):
+    IMPORT_FAILED = True
 
-def setUpModule():
-    raise unittest.SkipTest("External test dependencies like this seem bad.  Find a different "
-                            "built-in win32 API to use.")
 
+class Base(object):
```

Comment:
I think `BaseBindTest` is better name to explain its content. `Base` sounds too general.

Reply:
OK, I fixed it.

Reply:
And I removed `@unittest.skip("There is difference of ...` from `Test_LateBind`.

`CreateObject(..., dynamic=True)` and `xl.Range["A1", "C1"].Value[()] = (10,"20",31.4)` does not set `(None, None, 10.0)` to `A1:C1` even if Python >= 3.8.x(see https://github.com/enthought/comtypes/issues/212#issuecomment-1163176126).